src/App.js

```diff
@@ -2,10 +2,17 @@ import React, { Component } from 'react'
 import './App.css'
 
 class App extends Component{
+  constructor(props){
+    super(props)
+    this.state = {
+      currentCohort : "Echo"
```

Comment:
Hey I think you misspelled Echo-- it should be spelled SQUAD

Reply:
SQAUD 😁!!!